stac_api/runtime/src/config.py

```diff
@@ -78,6 +83,12 @@ def load_postgres_settings(self) -> "Settings":
         else:
             return Settings()
 
+    def router_prefix(self):
+        if self.path_prefix:
+            return self.path_prefix
+        else:
+            return ""
+
```

Comment:
Still looking for feedback on this approach. My thought was that if someone kept the ENV variable but included `None` instead of an empty string. That might be more of an issue in secrets manager...

Reply:
I was thinking we wanted to support defining the path prefix (or not) conditionally with a a configuration property but now that I've seen it both ways I could go either way. I also hadn't thought about the fact that the lambda environment only accepts string values.

Reply:
@smohiudd is someone includes "None", i'd imagine it would be a string not the python `None`, no?

Reply:
Yes I think you're right about that. This might have gotten a bit more complicated than needed. Will probably revert back to what you had.